package.json

```diff
@@ -22,22 +22,20 @@
     "postinstall": "lerna bootstrap",
     "start:all": "lerna run --parallel start",
     "start:docs": "docsify serve ./docs --port 4000",
-    "test": "jest --config .jest.json --coverage",
-    "test:all": "npm run test && lerna run --parallel test",
+    "test:all": "lerna run --parallel test",
     "test:related": "lerna run test:related",
     "deploy:ci": "lerna run deploy:ci",
     "deploy": "lerna run deploy"
   },
   "devDependencies": {
     "docsify-cli": "^4.4.2",
     "husky": "^4.3.8",
-    "jest": "26.6.3",
     "lerna": "^3.22.1"
   },
   "husky": {
     "hooks": {
       "pre-commit": "npm run lint:staged && npm run test:related",
-      "pre-push": "npm run lint:all && npm run test"
```

Comment:
`npm run test` no longer exists. It was previously only running the tests in `lib/tools.test.js`. Requiring this before pushing doesn't make much sense to me. Alternatively we could run `npm run test:all`, but this would mean a user has to have an active database server (and more?) in order to successfully push. Combined with the time taken to run all tests, this seems like an unnecessary overhead to run on `pre-push`.

Reply:
I agree with the change you did here